components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/java/org/wso2/carbon/identity/api/server/identity/governance/v1/core/ServerIdentityGovernanceService.java

```diff
@@ -302,6 +307,55 @@ public void updateGovernanceConnectorProperty(String categoryId, String connecto
         }
     }
 
+    /**
+     *
+     * Update multiple governance connector properties.
+     *
+     * @param categoryId                 Governance connector category id.
+     * @param multipleConnectorsPatchReq Governance connectors and properties to update.
+     */
+    public void updateGovernanceConnectorProperties(String categoryId,
+                                                    MultipleConnectorsPatchReq multipleConnectorsPatchReq) {
+
+        try {
+            IdentityGovernanceService identityGovernanceService = GovernanceDataHolder.getIdentityGovernanceService();
+            String tenantDomain = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain();
```

Comment:
Check whether we have any other way to obtain the tenant id as using thread local context is discouraged.

Reply:
The `ContextLoader.getTenantDomainFromContext()` [1] method can also be used to get the tenant ID. This method and the method I have used are the two ways in which the tenant domain has been obtained within the identity-api-server. 

Shall I use `ContextLoader.getTenantDomainFromContext()` [1]?

[1] https://github.com/wso2/identity-api-server/blob/master/components/org.wso2.carbon.identity.api.server.common/src/main/java/org/wso2/carbon/identity/api/server/common/ContextLoader.java#L53

Reply:
In side the ContextLoader also the tenant ID has been obtained from the thread local. Hence let's keep the method you used as it is.

Reply:
Ack